src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/401Tests.kt

```diff
@@ -4,21 +4,12 @@ import com.nhaarman.mockito_kotlin.doReturn
 import com.nhaarman.mockito_kotlin.mock
 import org.hamcrest.MatcherAssert.assertThat
 import org.hamcrest.Matchers.*
-import org.junit.ClassRule
-import org.junit.Test
-import org.vaccineimpact.orderlyweb.tests.unit_tests.templates.rules.FreemarkerTestRule
+import org.junit.jupiter.api.Test
 import org.vaccineimpact.orderlyweb.viewmodels.AppViewModel
 import org.xmlmatchers.XmlMatchers.hasXPath
 
-class _401Tests
+class _401Tests: FreeMarkerTest("401.ftl")
 {
-    companion object
-    {
-        @ClassRule
```

Comment:
class "rules" no longer exist, so now simply using a base class with a `BeforeEach` call on it (which actually seems simpler/preferable anyway)